apps/ledger-live-desktop/src/renderer/components/RecoverBanner/RecoverBanner.tsx

```diff
@@ -1,20 +1,24 @@
-import { Flex, Link, ProgressLoader, Text } from "@ledgerhq/react-ui";
+import { Flex, ProgressLoader, Text } from "@ledgerhq/react-ui";
 import React, { useCallback, useEffect, useMemo, useState } from "react";
 import { useHistory } from "react-router-dom";
 import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
 import { getStoreValue, setStoreValue } from "~/renderer/store";
 import { useCustomPath } from "@ledgerhq/live-common/hooks/recoverFeatureFlag";
 import { useTranslation } from "react-i18next";
-import styled from "styled-components";
 import { RecoverBannerType } from "./types";
-import { Card } from "~/renderer/components/Box";
-import ButtonV3 from "~/renderer/components/ButtonV3";
+import ActionCard from "../ContentCards/ActionCard";
+import { Card } from "../Box";
+import styled from "styled-components";
 
-const BannerContainer = styled(Card)`
+const Wrapper = styled(Card)`
   background-color: ${p => p.theme.colors.opacityPurple.c10};
+  margin: 20px 0px;
 `;
 
-export default function RecoverBanner() {
+/**
+ * @prop children: if a child is passed, it will be rendered instead of the default banner. this allows to do a passthrough to have first the recover banner, then the rest of the content.
+ */
+export default function RecoverBanner({ children }: { children?: React.ReactNode }) {
```

Comment:
You can use this type too if you want

```suggestion
export default function RecoverBanner({ children }: React.PropsWithChildren) {
```